docs/tasks/install_broker.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Install a Broker
+layout: docwithnav
+---
+
+A service broker must be added to the catalog before it can be used. This is
+done by creating either a ClusterServiceBroker or ServiceBroker object, and
+then allowing Service Catalog to sync with the broker. This is typically
+referred to as "registering" the broker.
+
```

Comment:
I think this is the right place to elaborate on the differences between Cluster Vs NS.  "A Service Broker can be registered with Service Catalog as either a Cluster Service Broker which makes its resources available cluster wide, to all users in all namespaces, or as a Service Broker which only exposes the resources within the specified namespace."